wunderio/nextjs-project
# PSW-33: npm updates

https://wunder.atlassian.net/browse/PSW-33

Changes proposed in this PR:
- Update npm packages
- Solve vulnerabilities

How to test:
1. Run "npm install" to get the packages updated
2. Run "npm run lint" to make sure that linting is working properly
